docs/quick-ingestion-guides/powerbi/configuration.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Configuration
+---
+# Configuring Your PowerBI Connector to DataHub
+
+Now that you have created a DataHub specific Azure AD app with the relevant access in [the prior step](setup.md), it's now time to set up a connection via the DataHub UI.
+
+## Configure Secrets
+
+1. Within DataHub, navigate to the **Ingestion** tab in the top, right corner of your screen
+
+<p align="center">
+  <img width="75%" alt="Navigate to the &quot;Ingestion Tab&quot;" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_ingestion_button.png"/>
+</p>
+
+:::note
+If you do not see the Ingestion tab, please contact your DataHub admin to grant you the correct permissions
+:::
+
+2. Navigate to the **Secrets** tab and click **Create new secret**
+
+<p align="center">
+   <img width="75%" alt="Secrets Tab" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_secrets_tab.png"/>
+</p>
+
+3. Create a client id secret
+
+  This will securely store your PowerBI `Application (client) ID` within DataHub
+
+   * Enter a name like `POWER_BI_CLIENT_ID` - we will use this later to refer to the `Application (client) ID`
+   * Enter the `Application (client) ID`
+   * Optionally add a description
+   * Click **Create**
+
+<p align="center">
+   <img width="70%" alt="Application (client) ID" src="https://raw.githubusercontent.com/datahub-project/static-assets-fork/main/imgs/guides/powerbi/powerbi-client-id-secret.png"/>
+</p>
+
+4. Create a secret to store Azure AD app secret
+
+  This will securely store your  Azure AD app secret
```

Comment:
Let's call this "client secret" everywhere - seems that Azure calls it that.

Reply:
done